src/VisualStudio/Core/Def/Implementation/ProjectSystem/FileChangeWatcher.cs

```diff
@@ -180,7 +182,7 @@ public Context(FileChangeWatcher fileChangeWatcher, ImmutableArray<WatchedDirect
                 foreach (var watchedDirectory in watchedDirectories)
                 {
                     _fileChangeWatcher.EnqueueWork(
-                        async service =>
+                        async (service, cancellationToken) =>
```

Comment:
➡️ Cancellation support removed

Reply:
Actually, hmm, why not cancel on our IThreadingContext disposal token? Because now that I look at it, this is totally violating this running things well after stuff might be shutting down...?

Reply:
It's not clear to me what the right thing to do here is.